libs/pages/services/src/lib/feature/page-helm-create-feature/page-helm-create-feature.tsx

```diff
@@ -0,0 +1,76 @@
+import { type GitProviderEnum } from 'qovery-typescript-axios'
+import { createContext, useContext, useState } from 'react'
+import { type UseFormReturn, useForm } from 'react-hook-form'
+import { Route, Routes, useNavigate, useParams } from 'react-router-dom'
+import { SERVICES_URL } from '@qovery/shared/routes'
+import { FunnelFlow } from '@qovery/shared/ui'
+import { ROUTER_SERVICE_HELM_CREATION } from '../../router/router'
+
+export const steps: { title: string }[] = [{ title: 'General data' }, { title: 'Values' }, { title: 'Summary' }]
+
+export type HelmGeneralData = {
```

Comment:
Can you extends existing Helm type from API ?

Reply:
I started to do that first, but finally, I can't because it's only for the general section and I reuse the `git` component for repository, branch, etc (with the same `name`), and it's difficult to manage without particular case for new `name`, it should be simpler to do that!